updateVersion.sh

```diff
@@ -70,18 +70,16 @@ fi
 # Set default behavior
 [ $P_RELEASE -eq 0 ] && P_INCREMENT=1
 
-. $VERSION_FILE
-
 # Increment build number
 if [ $P_INCREMENT -ne 0 ]; then
-    VERSION_OLD=$OMS_BUILDVERSION_BUILDNR
-    DATE_OLD=$OMS_BUILDVERSION_DATE
+    VERSION_OLD=`grep '^[A-Z]*_BUILDVERSION_BUILDNR' $VERSION_FILE | cut -d= -f2`
+    DATE_OLD=`grep '^[A-Z]*_BUILDVERSION_DATE' $VERSION_FILE | cut -d= -f2`
 
     VERSION_NEW=$(( $VERSION_OLD + 1 ))
     DATE_NEW=`date +%Y%m%d`
```

Comment:
I'm hesitant to do that without investigating all the places that this is used. If the usage doesn't expect that, it may cause issues.

Since this is generic, let's keep it as is for now, but we can certainly look at that once we're live with everything on GitHub.